copylibs.sh

```diff
@@ -0,0 +1,16 @@
+#/bin/bash
+set -e
+
+ldd /bin/clamscan /bin/freshclam /bin/ld | grep so | sed -e '/^[^\t]/ d' | sed -e 's/\t//' | sed -e 's/.*=..//' | sed -e 's/ (0.*)//' | sort | uniq | while read LIB;
```

Comment:
The idea here is to ask the executables what files they need and copy all of them with no regard for if those files will already exist in lambda or not. This should be much more future proof than listing out each lib not present in the base lambda image.